tests/foreman/api/test_user.py

```diff
@@ -193,7 +193,14 @@ def setUpClass(cls):
         """Create two roles and fetch the 'Anonymous' role."""
         super(UserRoleTestCase, cls).setUpClass()
         cls.roles = [entities.Role().create() for _ in range(2)]
-        roles = entities.Role().search(query={'search': 'name="Anonymous"'})
+        roles = entities.Role().search(query={'search': 'name="Default role"'})
+        if not roles:
+            # This is here because upstream changed name from "Anonumous" to
+            # "Default role" : Default role
+            # This can be removed once it get on all downstream versions
```

Comment:
@renzon perhaps its better to say here "This can be removed once this is in all downstream versions"